activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/service/ModelService.java

```diff
@@ -103,8 +111,28 @@ public ModelService(ModelRepository modelRepository,
                                          pageable);
     }
 
+    public Model createNewModelObject(String type,
+                                      String name) {
+      Model model = newModelInstance();
+      model.setType(type);
+      model.setName(name);
+      return model;
+    }
+
+    public Model newModelInstance() {
+      try {
+        return (Model) modelRepository.getModelType().getConstructor().newInstance();
+      } catch (InstantiationException |
+        IllegalAccessException |
+        NoSuchMethodException |
+        InvocationTargetException e) {
+        throw new RuntimeException(e);
```

Comment:
should we create a dedicated `ImportException` or `ProjectImportException` instead of `RuntimeException`
so we can catch the new exception more easily in the Controller exception handlers and return the appropriate response code